src/Table.tsx

```diff
@@ -186,6 +187,7 @@ export function useDatatableLifecycle({
 
   let data = tableBody;
   let maxPage;
+  let totalNumberOfResults = tableBody.length;
```

Comment:
Nit: since `totalNumberOfResults` won't change, let's use `const`?

```suggestion
  const totalNumberOfResults = tableBody.length;
```